.infrastructure/50_storage.tf

```
@@ -57,8 +57,7 @@ module "dynamodb_webinar_questions" {
   name                           = "WebinarQuestions"
   hash_key                       = "webinarId"
   range_key                      = "createdAt"
-  ttl_enabled                    = true
-  ttl_attribute_name             = "expireAt"
+  ttl_enabled                    = false
```

We have to do a two step deploy, step one to disable the `ttl`, step two to remove it (means two PR)

```suggestion
  ttl_enabled                    = false
  ttl_attribute_name             = "expireAt"
```